credoai/evaluators/utils/validation.py

```diff
@@ -29,19 +38,19 @@ def check_model_instance(obj, inst_type, name="Model"):
 
 
 def check_feature_presence(feature_name, df, name):
-    if isinstance(df, DataFrame):
+    if isinstance(df, pd.DataFrame):
         if not feature_name in df.columns:
             message = f"Feature {feature_name} not found in dataframe {name}"
             raise ValidationError(message)
-    if isinstance(df, Series):
+    if isinstance(df, pd.Series):
         if not df.name == feature_name:
             message = f"Feature {feature_name} not found in series {name}"
             raise ValidationError(message)
 
 
 def check_existence(obj, name=None):
```

Comment:
I'm confused about what this function is for. Why the pandas check? (I know you didn't write it though)

Reply:
I actually have no clue. It may be that this is supposed to check for emptiness?

If so, inside the pandas if-statement, we can use the `.empty` attribute.
The other if-statement is assuming any other provided object follows the semantics of built-in python data types where emptiness can be checked with a simple `not`. Not sure if this is liable to fail for any non-standard library data types we use --> should be fine for Tensors, at least, since these need to be initialized with some value (this is driven by the fact that tensorflow is backed by C for speed and so the tensors are immutable once instantiated).

Reply:
Updated based on your suggestion.